app/views/reviews/edit.html.erb

```diff
@@ -0,0 +1,20 @@
+<% flash.each do |type, msg| %>
+  <div>
+    <font size="5" color="red"><%= msg %></font>
+  </div>
+<% end %>
+
+<h2>Fill out this form to update a shelter review!</h2>
+<%= form_tag "/reviews/#{@review.id}", method: 'PATCH' do %>
+    <%= label_tag :title %>
+    <%= text_field_tag :title, @review.title%><br>
+    <%= label_tag :rating %>
+    <%= text_field_tag :rating, @review.rating %><br>
+    <%= label_tag :content %>
+    <%= text_field_tag :content, @review.content %><br>
+    <%= label_tag :picture %>
+    <%= text_field_tag :picture, @review.picture %><br>
```

Comment:
The `@review.___` following the text_field_tag allows the initial entries to be prefilled in this edit form.

Reply:
I like that add. Solid.